react/react1/week1/todo-list/src/components/toDoList.js

```diff
@@ -0,0 +1,20 @@
+import todos from "../data/todos";
+import ToDoItem from "./toDoItem";
+
+export default function ToDoList() {
```

Comment:
Minor detail ->
Your component name is ToDoList but the file name is toDoList.
Try to keep it consistent and rename the file as well to ToDoList.js
Same with ToDoItem.js